source/Halibut.Tests/Support/TestAttributes/LatestAndPreviousClientAndServiceVersionsTestCasesAttribute.cs

```diff
@@ -56,6 +56,7 @@ public static IEnumerator<ClientAndServiceTestCase> GetEnumerator(bool testWebSo
                         ClientAndServiceTestVersion.Latest(),
                         ClientAndServiceTestVersion.ClientOfVersion(PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417.ClientVersion),
                         ClientAndServiceTestVersion.ServiceOfVersion(PreviousVersions.v5_0_236_Used_In_Tentacle_6_3_417.ServiceVersion),
+                        ClientAndServiceTestVersion.ServiceOfVersion(PreviousVersions.v4_4_8.ServiceVersion),
```

Comment:
I don't think we should generally test on this version since it is unstable.

What is the minimum amount of testing against this version we can get away with that will likely prevent a regression?

Reply:
As `4.4.8` seems stable with net48 and running tests on net48 showed the issues that caused this fix I've one with 

- Run net48 tests against Halibut Service 4.4.8
- Don't run net6.0 tests against Halibut Service 4.4.8

I've used the version redirects to achieve this 

```
#if NETFRAMEWORK
        public static readonly HalibutVersions v4_4_8 = new(new Version("4.4.8"), pollingOverWebSocketServiceVersion: v6_0_658_WebSocket_Stability_Fixes);
#else
        // A net6.0 client causes an older halibut service without buffer over-read changes to be prone to hanging or erroring when reading the stream.
        // This instability means we cannot really have automated tests for net6.0 and older these older Halibut Services
        public static readonly HalibutVersions v4_4_8 = new(new Version("4.4.8"), pollingOverWebSocketServiceVersion: v6_0_658_WebSocket_Stability_Fixes, pollingServiceVersion: v5_0_236_Stable_Streams, listeningServiceVersion: v5_0_236_Stable_Streams);
#endif
```

Reply:
This has gone 🟢 on Team City

Reply:
I'll run a few more builds and make sure it's stable before merging

Reply:
Yeah all of the overreads or expected overreads "happen" to work when both sides are `net4.8`, so this should be safe so long as nothing OS specific comes into play and the framework doesn't change.